public/editor/scripts/project/project.js

```diff
@@ -237,22 +238,48 @@ define(function(require) {
             return callback(err);
           }
 
-          // Find an HTML file to open in the project, hopefully /index.html
-          var sh = new _fs.Shell();
-          sh.find(getRoot(), {name: "*.html"}, function(err, found) {
+          // Find the index.html file in the project root to open
+          _fs.readdir(getRoot(), function(err, found) {
```

Comment:
I'm not thinking clearly...the right thing to do here is `_fs.exists(Path.join(getRoot(), "index.html"), function(exists) {` which will replace all the variables and your loop, and just report on whether this file is there.